openedge-plugin/src/main/java/org/sonar/plugins/openedge/foundation/OpenEdgeComponents.java

```diff
@@ -192,21 +176,26 @@ public IIdProvider getIdProvider() {
     return Collections.unmodifiableMap(dfChecksMap);
   }
 
-  public Licence getLicence(String repoName) {
-    return licences.get(repoName);
+  public Licence getLicence(String repoName, String permId) {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this unused method parameter "repoName". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar.riverside-software.fr/coding_rules#rule_key=squid%3AS1172)